From 755d544da685e97fcaa60f536838eeff194d6432 Mon Sep 17 00:00:00 2001
From: JeongBong Seo <jb.seo@lge.com>
Date: Wed, 3 Apr 2019 22:28:40 +0900
Subject: Fix configure error for python

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d93e3d2..0d162a0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,7 +21,7 @@ if(NOT ANDROID)
     if(Boost_VERSION LESS 106500)
       find_package(Boost REQUIRED python)
     else()
-      find_package(Boost REQUIRED python3)
+      find_package(Boost REQUIRED python35)
     endif()
   endif()
 else()
-- 
2.17.1

